src/one_dragon/base/controller/pc_controller_base.py

```
@@ -519,26 +522,33 @@ def mouse_move(self, game_pos: Point) -> None:
     def center_point(self) -> Point:
         return Point(self.standard_width // 2, self.standard_height // 2)
 
+    def win_click(self, pos: Point = None, press_time: float = 0, primary: bool = True, low_delay: bool = False):
+        """点击鼠标。
+
+        Args:
+            pos: 屏幕坐标
+            press_time: 按住时间
+            primary: 是否点击鼠标主要按键（通常是左键）
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)